packages/tools/viewer/src/viewer.ts

```diff
@@ -1102,6 +1154,73 @@ export class Viewer implements IDisposable {
         return true;
     }
 
+    /**
+     * Refresh bounding info to ensure morph target and skeletal animations are taken into account.
+     * @param model The AssetContainer to refresh.
+     */
+    private _refreshBoundingInfo(model: AssetContainer): void {
+        model.meshes.forEach((mesh) => {
+            let cache = this._meshDataCache.get(mesh);
+            if (!cache) {
+                cache = {};
+                this._meshDataCache.set(mesh, cache);
+            }
+            mesh.refreshBoundingInfo({ applyMorph: true, applySkeleton: true, cache });
+        });
+    }
+
+    /**
+     * Calculates the alpha, beta, and radius along with the target point.
+     * The target point is determined based on the camera's forward ray:
+     *   - If an intersection with the main model is found, the first hit point is used as the target.
+     *   - If no intersection is detected, a fallback target is calculated by projecting
+     *     the distance between the camera and the main model's center along the forward ray.
+     *
+     * @param camera The reference camera used to computes infos
+     * @param predicate Used to define predicate for selecting meshes and instances (if exist)
+     * @returns An object containing the alpha, beta, radius and target properties, or null if no model found
+     */
+    public async getArcRotateCameraInfos(camera: Camera, predicate?: MeshPredicate): Promise<Nullable<ViewerArcRotateCameraInfos>> {

```

Comment:
As we discussed a bit on the side, can we move this now to be a private function inside `viewerElement.ts` since `ViewerElement` will now have access to the bounding info through the model info?